src/components/backend-ai-settings-view.ts

```diff
@@ -432,28 +480,86 @@ export default class BackendAiSettingsView extends BackendAIPage {
     return true;
   }
 
+  _showDialog() {
+    const modifyEnvDialog = this.shadowRoot.querySelector('#env-dialog');
+    modifyEnvDialog.show();
+  }
+
+  _closeDialog() {
+    const modifyEnvDialog = this.shadowRoot.querySelector('#env-dialog');
+    modifyEnvDialog.hide();
+  }
+
+  _showConfigDescription(e, item) {
+    e.stopPropagation();
+    const schedulerConfigDescription = {
+      'default': {
+        'title': _text('settings.SchedulerDefault'),
+        'desc': _text('settings.SchedulerDefaultDescription')
+      },
+      'retries': {
+        'title': _text('settings.SessionCreationRetries'),
+        'desc': _text('settings.SessionCreationRetriesDescription') + _text('settings.FifoOnly')
+      }
+    };
+    if (item in schedulerConfigDescription) {
+      this._helpDescriptionTitle = schedulerConfigDescription[item].title;
+      this._helpDescription = schedulerConfigDescription[item].desc;
+      const desc = this.shadowRoot.querySelector('#help-description');
+      desc.show();
+    }
+  }
+
   /**
-   * Change Scheduler and notify.
-   *
-   * @param {HTMLElement} e - scheduler setting component
-   * */
-  changeScheduler(e) {
-    const scheduler = e.target.value;
-    if (['fifo', 'lifo', 'drf'].includes(scheduler)) {
-      const detail = {
-        'num_retries_to_skip': 0
-      };
-      globalThis.backendaiclient.setting.set(`plugins/scheduler/${scheduler}`, detail).then((response) => {
-        this.notification.text = _text('settings.JobSchedulerUpdated');
+   * Update etcd scheduler options(num_retries_to_skip) and close the dialog.
+   */
+  saveAndCloseDialog() {
+    let tempNumRetries = this.shadowRoot.querySelector('#num-retries').value;
+    tempNumRetries = tempNumRetries === '' ? '0' : tempNumRetries;
+    if (['fifo', 'lifo', 'drf'].includes(this.selectedSchedulerType) && tempNumRetries.match(/^[0-9]*$/)) {
+      // currently, only support when scheduler type is fifo.
+      if (this.selectedSchedulerType === 'fifo' || (this.selectedSchedulerType !== 'fifo' && tempNumRetries === '0')) {
+        const numRetries = parseInt(tempNumRetries).toString();
+        globalThis.backendaiclient.setting.set(`plugins/scheduler/${this.selectedSchedulerType}/num_retries_to_skip`, numRetries)
```

Comment:
How about generalizing features per each scheduler type?
In this code, the user can update `num_retries_to_skip` key and value only.
As we have a high chance of applying multiple keys and values, Let's make it more general,
such as sending as JSON below.
`{ num_retries_to_skip :  ... , new_feature: ...,}`